### PR TITLE
[tools][generator] Added logic for non standard planet.

### PR DESCRIPTION
**Pull request by maksimandrianov**

### CODE REVIEW

File: tools/python/maps_generator/__main__.py [hunk 1]
```diff
@@ -220,6 +221,11 @@ def compare(a, b):
         raise SkipError(f"Stages {set(options['skip']) - set(ALL_STAGES)} "
                         f"not found.")
 
+    if settings.PLANET_URL != settings.DEFAULT_PLANET_URL:
+        options["skip"] += stages_as_string(
+            stage_update_planet
```
Comment (gmoryes) [inline tools/python/maps_generator/__main__.py]: мб без переноса?

Comment (gmoryes) [reply tools/python/maps_generator/__main__.py]: `options["skip"] += stages_as_string(stage_update_planet)`

File: tools/python/maps_generator/maps_generator.py [hunk 2]
```diff
@@ -38,8 +38,10 @@ def download_external(url_to_path: dict):
 
 @stage
 def stage_download_and_convert_planet(env, **kwargs):
-    if not is_verified(settings.PLANET_O5M):
-        stages.stage_download_and_convert_planet(env, **kwargs)
+    force_update = not env.is_accepted_stage(stage_update_planet)
```
Comment (gmoryes) [inline tools/python/maps_generator/maps_generator.py]: force_download?

**Comment by mpimenov** [comment]
s/standart/standard/g

File: tools/python/maps_generator/maps_generator.py (hunk 2 above)
Comment (maksimandrianov) [reply tools/python/maps_generator/maps_generator.py]: ++

File: tools/python/maps_generator/__main__.py (hunk 1 above)
Comment (maksimandrianov) [reply tools/python/maps_generator/__main__.py]: ++

**Comment by maksimandrianov** [comment]
> s/standart/standard/g

fixed

**Pull request merged**